doc/release-notes/5.0-release-notes.md

```diff
@@ -0,0 +1,313 @@
+
+### The setting :PIDAsynchRegFileCount is deprecated as of v5.0
+
+It used to specify the number of datafiles in the dataset to warrant adding a lock during publishing. As of v5.0 all datasets get locked for the duration of the publishing process. The setting will be ignored if present.
+
+### Location Changes for Related Projects
+
+The dataverse-ansible and dataverse-previewers repositories have been moved to the GDCC Organization on GitHub. If you have been referencing the dataverse-ansible repository from IQSS and the dataverse-previewers from QDR, please instead use them from their new locations:
+
+<https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible>
+<https://github.com/GlobalDataverseCommunityConsortium/dataverse-previewers>
+
+### Harvesting Improvements
+
+Many updates have been made to address common Harvesting failures. You may see Harvests complete more often and have a higher success rate on a dataset-by-dataset basis.
+
+### New JVM Options and Database Settings
+
+Several new JVM options and DB Settings have been added in this release. More documentation about each of these settings can be found in the Configuration section of the [Installation Guide](http://guides.dataverse.org/en/5.0/installation/config.rst).
+
+#### New JVM Options
+
+- :doi.baseurlstringnext: Set with a value of "https://api.datacite.org" for production environments and "https://api.test.datacite.org" for test environments. Must be set if you are using DataCite as your DOI provider.
+- :dataverse.useripaddresssourceheader: If set, specifies an HTTP Header such as X-Forwarded-For to use to retrieve the user's IP address. This setting is useful in cases such as running Dataverse behind load balancers where the default option of getting the Remote Address from the servlet isn't correct (e.g. it would be the load balancer IP address). Note that unless your installation always sets the header you configure here, this could be used as a way to spoof the user's address.
+- :FileValidationOnPublishEnabled: Toggles validation of the physical files in the dataset when it's published, by recalculating the checksums and comparing against the values stored in the DataFile table. By default this setting is absent and Dataverse assumes it to be true. If enabled, the validation will be performed asynchronously, similarly to how we handle assigning persistent identifiers to datafiles, with the dataset locked for the duration of the publishing process.
```

Comment:
:FileValidationOnPublishEnabled is a database setting, not a JVM option, so it should be moved down there.